### ISSUE TITLE
Setting `ignore-unfixed` no longer working with `0.0.9`

### ISSUE THREAD

**Issue by nathankleyn** [aquasecurity/trivy-action#30]
Hello! 👋 

We are using your wonderful action to scan for CVEs in our Docker images — thank you for hard work and contribution to open-source!

Sadly we noticed that there seems to be a regression in `0.0.9` with respect to `ignore-unfixed`.

In `0.0.8` setting `ignore-unfixed: true` correctly ignores CVEs that have no published fix version yet.
In `0.0.9` setting `ignore-unfixed: true` has no effect and we can see CVEs being failed on that have no published fix version.

For example:

```
========================================================================================
Total: 44 (HIGH: 40, CRITICAL: 4)

+----------------------+------------------+----------+---------------------------+---------------+--------------------------------------------------------------+
|       LIBRARY        | VULNERABILITY ID | SEVERITY |     INSTALLED VERSION     | FIXED VERSION |                            TITLE                             |
+----------------------+------------------+----------+---------------------------+---------------+--------------------------------------------------------------+
| curl                 | CVE-2020-8169    | HIGH     | 7.64.0-4+deb10u1          |               | libcurl: partial password                                    |
|                      |                  |          |                           |               | leak over DNS on HTTP redirect                               |
|                      |                  |          |                           |               | -->avd.aquasec.com/nvd/cve-2020-8169                         |
+                      +------------------+          +                           +---------------+--------------------------------------------------------------+
```

This is a scan result from 0.0.9 with `ignore-unfixed: true` on and we can clearly see a CVE for `curl` that does not yet have a fix version published.

Here is the action step config we were using:

```
      - name: Run Trivy vulnerability scanner
        uses: aquasecurity/trivy-action@master
        with:
          image-ref: "<docker image repo/tag here>"
          format: "table"
          exit-code: "1"
          ignore-unfixed: true
          severity: "CRITICAL,HIGH"
```

Changing from `master` to `0.0.8` after `0.0.9` was published fixed the issue:

```
      - name: Run Trivy vulnerability scanner
        uses: aquasecurity/trivy-action@0.0.8
        with:
          image-ref: "<docker image repo/tag here>"
          format: "table"
          exit-code: "1"
          ignore-unfixed: true
          severity: "CRITICAL,HIGH"
```

**Comment by nathankleyn** [comment]
I'm so sorry, should have checed first but just seen you've published 0.0.10 and it has a fix for this!

This issue was related to https://github.com/aquasecurity/trivy-action/issues/28. Closing!

Thanks again for everything!

**Comment by domgraziano** [comment]
> I'm so sorry, should have checked again before raising! Just seen you've published 0.0.10 and it has a fix for this! 😂 🤦
> This issue was related to #28. Closing!
> 
> Thanks again for everything!

note that the version has been published as v0.0.10 instead of simply 0.0.10 as all the previous version this could cause quite a lot of confusion, I'd suggest to create a tag with 0.0.10 without deleting v0.0.10 as someone it might have used that already

**Comment by domgraziano** [comment]
> > I'm so sorry, should have checked again before raising! Just seen you've published 0.0.10 and it has a fix for this! 😂 🤦
> > This issue was related to #28. Closing!
> > Thanks again for everything!
> 
> note that the version has been published as v0.0.10 instead of simply 0.0.10 as all the previous version this could cause quite a lot of confusion, I'd suggest to create a tag with 0.0.10 without deleting v0.0.10 as someone it might have used that already

I'd also suggest to delete 0.0.9 and recreate it from master (without caring about the fact that is identical to 0.0.10), that would automatically fix issues for other people trying to checkout 0.0.9 which holds the bug in it
